### PR TITLE
new date format for market end time, added date and full date time he…

### PR DESCRIPTION
**Pull request by bthaile**
…lper methods

new simple date format for market end time. New helper for full market date as well.

image

### CODE REVIEW

File: packages/augur-simplified/src/utils/date-utils.ts [hunk 1]
```diff
@@ -30,3 +30,19 @@ export const getTimeFormat = (timestamp) => {
...
+  const date = new Date(inMilli)
+  const year = date.getFullYear();
+  const monthDay = getDayFormat(timestamp);
+  return `${monthDay} ${year}`
+}
+
+export const fullMarketEndtime = (timestamp: string | number) => {
+  if (!timestamp) return 'Missing';
+  // use existing to make sure to be consistent
+  const monthDay = getMarketEndtimeDate(timestamp);
```
Comment (bconfortin) [inline packages/augur-simplified/src/utils/date-utils.ts]: monthDay in `getMarketEndtimeDate` returns `${mon} ${day}`
monthDay in `fullMarketEndtime` returns `${monthDay} ${year}`

Is that correct? Shouldn't at least the variable names be different?

Comment (bthaile) [reply packages/augur-simplified/src/utils/date-utils.ts]: sure. I'll make more clear

File: packages/augur-simplified/src/utils/date-utils.ts [hunk 2]
```diff
@@ -30,3 +30,19 @@ export const getTimeFormat = (timestamp) => {
...
+  const date = new Date(inMilli)
+  const year = date.getFullYear();
+  const monthDay = getDayFormat(timestamp);
+  return `${monthDay} ${year}`
+}
+
+export const getMarketEndtimeFull = (timestamp: string | number) => {
+  if (!timestamp) return 'Missing';
+  // use existing to make sure to be consistent
+  const monthDay = getMarketEndtimeDate(timestamp);
```
Comment (bconfortin) [inline packages/augur-simplified/src/utils/date-utils.ts]: Variable is named `monthDay` yet it represents `monthDay and year`

**Review by bconfortin — CHANGES_REQUESTED** [review]

File: packages/augur-simplified/src/utils/date-utils.ts (hunk 2 above)
Comment (bthaile) [reply packages/augur-simplified/src/utils/date-utils.ts]: updated

**Review by bconfortin — APPROVED** [review]
Lgtm

**Pull request merged**